src/formats/prometheus_text_format.erl

```diff
@@ -105,10 +109,12 @@ emit_series(Fd, Name, Labels, Value) ->
 escape_metric_help(Help) ->
   sub(sub(Help, "\\", "\\\\\\\\"), "\n", "\\\\n").
 
+%% FIXME: -spec escape_label_value(term()) -> string().
 escape_label_value(LValue) when is_list(LValue)->
   sub(sub(sub(LValue, "\\", "\\\\\\\\"), "\n", "\\\\n"), "\"", "\\\\\"");
 escape_label_value(LValue) when is_binary(LValue) ->
   escape_label_value(binary_to_list(LValue));
+%% FIXME: dialyzer says this "can never match"
```

Comment:
Oh this was trickier than expected. `infinity` has to be used because this atom represents infinity for gpb. As usual one more indirection hopefully solved the problem: https://github.com/deadtrickster/prometheus.erl/commit/89a3f3505311b5ecca65ba628785df4acc1985a2.